src/meta/meta_ckpt.c

```diff
@@ -1072,7 +1091,7 @@ __wt_metadata_correct_base_write_gen(WT_SESSION_IMPL *session)
         WT_ERR(cursor->get_value(cursor, &config));
 
         /* Update base write gen to the write gen. */
```

Comment:
Same here. Outdated comment.

Reply:
Fixed in [04de73c](https://github.com/wiredtiger/wiredtiger/pull/8245/commits/04de73cd3345c280490cd5e336ac77226b5c37ed)